R/assertions.R

```diff
@@ -1448,3 +1448,49 @@ assert_date_var <- function(dataset, var, dataset_name = NULL, var_name = NULL)
     ))
   }
 }
+
+#' Is a Variable vector is a Date or Datetime Variable?
+#'
+#' Checks if a variable vector is a date or datetime variable
```

Comment:
This needs some wordsmithing.  Also, should we help developers differentiate between `assert_date_var` and `assert_date_vector` as I got confused with the titles of these two functions and their purpose.

Reply:
![image](https://user-images.githubusercontent.com/10111024/197272944-1d28e1fb-3190-4c06-b546-d7630e0eaded.png)

Reply:
Beautiful much clearer to me!

Reply:
Great, updates implemented!